spec/index.html

```diff
@@ -7410,6 +7410,114 @@ <h5>TZ</h5>
+            </p>
+            <pre class="query nohighlight">
+              PREFIX : <http://example/>
+              SELECT ?s ?date {
+                  ?s ?p ?o
+                  BIND( &lt;&lt; ?s ?p ?o &gt;&gt; AS ?qt )
+                  ?qt :tripleAdded ?date
+              }
+            </pre>
+            <pre class="query nohighlight">
+              PREFIX : <http://example/>
+              SELECT ?s ?date {
+                  ?s ?p ?o
+                  BIND( TRIPLE(?s, ?p, ?o) AS ?qt )
+                  ?qt :tripleAdded ?date
+              }
+            </pre>
+          </section>
+
+          <section id="func-subject">
+            <h5>SUBJECT</h5>
+            <pre class="prototype nohighlight"><span class="return">RDF term</span>  <span class="operator">SUBJECT</span> (<span class="type RDFterm">quoted triple</span> <span class="name">quoted-triple</span>)</pre>
+            <p>
+              If <code style="color:black">triple</code> is a 
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>, 
+              the function returns the subject of the quoted triple. 
+              If the argument is not a <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              an error is raised.
+            </p>
+          </section>
+          
+          <section id="func-predicate">
+            <h5>PREDICATE</h5>
+            <pre class="prototype nohighlight"><span class="return">RDF term</span>  <span class="operator">PREDICATE</span> (<span class="type RDFterm">quoted triple</span> <span class="name">quoted-triple</span>)</pre>
+            <p>
+              If <code style="color:black">triple</code> is a 
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              the function returns the predicate of the quoted triple.
+              If the argument is not a <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              an error is raised.
+            </p>
+          </section>
+
+          <section id="func-object">
+            <h5>OBJECT</h5>
+            <pre class="prototype nohighlight"><span class="return">RDF term</span>  <span class="operator">OBJECT</span> (<span class="type RDFterm">quoted triple</span> <span class="name">quoted-triple</span>)</pre>
+            <p>
+              If <code style="color:black">triple</code> is a <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              the function returns the object of the quoted triple.
+              If the argument is not a <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>, 
+              an error is raised.
+            </p>
+          </section>
+
+          <section>
+            <h5>isTRIPLE</h5>
+            <pre class="prototype nohighlight"><span class="return">xsd:boolean</span>  <span class="operator">isTRIPLE</span> (<span class="type RDFterm">RDF term</span>)</pre>
```

Comment:
For the sake of consistency with the function signatures in the previous subsections.
```suggestion
            <pre class="prototype nohighlight"><span class="return">xsd:boolean</span>  <span class="operator">isTRIPLE</span> (<span class="type RDFterm">RDF term</span> <span class="name">term</span>)</pre>
```